### ISSUE TITLE
Allow download resuming

### ISSUE THREAD

**Issue by JAJames** [TotemArts/Rx_Launcher#1]
As a user, I want to be able to pause and potentially shutdown the Renegade X launcher and resume downloading at a later time

- [ ] Pause/resume downloads
- [] Abort an existing download
- [] Empty cache of download and restart download

**Comment by AlienXAXS** [comment]
Just as a note here:
Downloads will resume from where they left off already if the patch\download folder contains items, i am however unsure if the launcher does any hash checking on this which will be needed.

ToDo: Check if the already existing items are subject to hashing checks, if not implement this.

**Comment by AlienXAXS** [comment]
This works with #12 when the user cancels the download and starts it again later - however an extra dialog perhaps upon pressing Cancel to state "Are you sure?" and "You can resume this download later" might be useful to the end user.

**Comment by JAJames** [comment]
With those changes in place, I'll rephrase this to focus on resolving the bug where resumed download files are checked and accounted for in the download phase rather than the verification phase.
